addons/survey/data/mail_templates_email_layouts.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="survey_mail_notification_layout"
+                  name="Mail: mail notification layout for survey"
+                  inherit_id="mail.mail_notification_layout" primary="True">
+            <xpath expr="//body/div/div[2]/table/tbody/tr[3]/td/table/tbody/tr/td[1]
```

Comment:
Please use a proper implementation with a clean override of "_notify_get_recipients_groups", this xpath is way too complex and random to be robust.

And even then the correct way to do it is to use "_notify_get_recipients_groups", not a custom layout template.

Reply:
For this part instead of using this complex xpath i had just passed the id on that target element and used it here not sure if it is the correct way or not but as i can see that survey is not linked up with the 'mail.thread' so i was not able to override the _notify_get_recipients_groups. or am I missing something?